analyzer/codechecker_analyzer/cmd/check.py

```diff
@@ -767,6 +767,14 @@ def __update_if_key_exists(source, target, key):
             LOG.debug("Calling LOG with args:")
             LOG.debug(log_args)
 
+            # If not explicitly given the debug log file of ld_logger is placed
+            # in report directory if any. Otherwise parallel "CodeChecker
+            # check" commands would overwrite each other's log files under /tmp
+            # which is the default location for "CodeChecker check".
+            if 'CC_LOGGER_DEBUG_FILE' not in os.environ:
```

Comment:
At the `CodeChecker log` command in the help message we mention this variable. It would be good to mention at the `CodeChecker check` command too and highlight the information that if this environment variable is not specified it will insert this debug log file to the report directory.